sdk/identity/Azure.Identity/src/ProcessRunner.cs

```diff
@@ -26,10 +27,17 @@ internal sealed class ProcessRunner : IDisposable
         public int ExitCode => _process.ExitCode;
 
         public ProcessRunner(IProcess process, TimeSpan timeout, bool logPII, CancellationToken cancellationToken)
+            : this(process, timeout, logPII, false, cancellationToken)
+        {
+            // This constructor delegates to the main one with default redirectStandardInput = false
+        }
```

Comment:
no need to add this commend as it is implied
```suggestion
        { }
```